evap/results/exporters.py

```diff
@@ -285,12 +323,17 @@ def write_questionnaire(self, questionnaire, evaluations_with_results, contribut
 
     # pylint: disable=arguments-differ
     def export_impl(
-        self, semesters, selection_list, include_not_enough_voters=False, include_unpublished=False, contributor=None
-    ):
+        self,
+        semesters: Sequence[Semester],
+        selection_list,
```

Comment:
Seems to be a sequence of `(degree_ids, course_type_ids)` pairs in the code below -> `Sequence[tuple[Iterable[int], Iterable[int]]]`?